docs/conf.py

```diff
@@ -44,6 +44,26 @@
 # The full version, including alpha/beta/rc tags
 release = rockcraft.__version__
 
+# Update with the favicon for your product
+html_favicon = "_static/favicon.png"
+
+html_context = {
+    # Change to the discourse instance you want to be able to link to
+    # (use an empty value if you don't want to link)
+    "discourse": "https://discourse.ubuntu.com/c/rocks/117",
+    # Change to the GitHub info for your project
+    "github_url": "https://github.com/canonical/rockcraft",
+    # Change to the branch for this version of the documentation
+    "github_version": "main",
```

Comment:
doesn' the readthedocs infra handle this for us?

Reply:
This is needed for having the link for discourse and GitHub on the RTD

```
* ``discourse_prefix`` - the URL to the Discourse instance your project uses (needed to add links to posts using the ``:discourse:`` metadata at the top of a file)
* ``github_url`` - the link to your GitHub repository (needed to create the Edit link in the footer and the feedback button)
* ``github_version`` - the branch that contains this version of the documentation
* ``github_folder`` - the folder that contains the documentation files
```